src/cmakeFindModules/FindOpenImageIO.cmake

```diff
@@ -0,0 +1,18 @@
+#find headers
```

Comment:
Why introduce yet another dependency instead of supporting only the formats that OpenCV supports?